tools/dep_updaters/update-base64.sh

```diff
@@ -51,6 +51,10 @@ mv "$WORKSPACE/base64" "$DEPS_DIR/base64/"
 # to work, we create it and leave it empty.
 echo "// Intentionally empty" >> "$DEPS_DIR/base64/base64/lib/config.h"
 
+# Clear out .gitignore, otherwise config.h is ignored. That's dangerous when
+# people check in our tarballs into source control and run `git clean`.
+echo "# Intentionally empty" >> "$DEPS_DIR/base64/base64/.gitignore"
```

Comment:
```suggestion
echo "# Intentionally empty" > "$DEPS_DIR/base64/base64/.gitignore"
```

Otherwise the file is not cleared out, no?

Reply:
Yeah... I also changed it on line 52.